README.md

```diff
@@ -1 +1,21 @@
-# pokedex-2018-honeybadgers
\ No newline at end of file
+# pokedex-2018-honeybadgers
+
+### Database Setup
+
+Step 1: Set database IP address
+
+In 'dataScrape/apiPull.php', line 7, where pokemonTableUpdate is instantiated, change the second argument to the IP address of your database.
```

Comment:
could we make this a constant in this file? easier to find then rather than an arbitrary argument

Reply:
changed in 9930963